### PR TITLE
Debugging in e2e

### PR DESCRIPTION
**Pull request by mjudeikis**
Seen multiple failures in our local E2E for this particular failure.
Need to get to the bottom of this why in 15 min logging stack is not ready.

### CODE REVIEW

File: test/e2e/operator.go [hunk 1]
```diff
@@ -131,7 +100,11 @@ var _ = Describe("ARO Operator - Geneva Logging", func() {
 		}
 
 		err := wait.PollImmediate(30*time.Second, 15*time.Minute, mdsdReady)
-		Expect(err).NotTo(HaveOccurred())
```
Comment (jim-minter) [inline test/e2e/operator.go:134]: don't drop this line or the tests will start passing when it should have failed.

File: test/e2e/utils.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
+package e2e
```
Comment (jim-minter) [inline test/e2e/utils.go:1]: utils.go is an anti-pattern, please don't do it.

File: test/e2e/utils.go [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+func dumpEvents(ctx context.Context, namespace string) error {
+	events, err := clients.Kubernetes.EventsV1().Events(namespace).List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	for _, event := range events.Items {
+		spew.Dump(event)
```
Comment (jim-minter) [inline test/e2e/utils.go:51]: use a logger.

**Review by jim-minter — CHANGES_REQUESTED** [review]

File: test/e2e/utils.go (hunk 3 above)
Comment (mjudeikis) [reply test/e2e/utils.go:51]: Events data structure is not very friendly for this, this is easier :/ . This is just temporary code until we can identify flakes reason. I can migrate it but I dont see strong reasons for something like this which is just debug code until we get stuff sorted.

File: test/e2e/utils.go (hunk 2 above)
Comment (mjudeikis) [reply test/e2e/utils.go:1]: i can rename it to helpers? We kinda already went this path all around the place.

File: test/e2e/operator.go [hunk 4]
```diff
@@ -132,6 +144,11 @@ var _ = Describe("ARO Operator - Geneva Logging", func() {
 
 		err := wait.PollImmediate(30*time.Second, 15*time.Minute, mdsdReady)
 		Expect(err).NotTo(HaveOccurred())
+		if err != nil {
```
Comment (jim-minter) [inline test/e2e/operator.go:147]: This doesn't work.

**Review by jim-minter — CHANGES_REQUESTED** [review]

File: test/e2e/operator.go (hunk 4 above)
Comment (mjudeikis) [reply test/e2e/operator.go:147]: Why this does not work? We get timeout errors in the tests where `mdsdReady` fails(?)

**Review by jim-minter — APPROVED** [review]
merge on green

**Pull request merged**